lib/kvao/iterate-keys.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var assert = require('assert');
+var utils = require('../utils');
+
+/**
+ * Asynchronously iterate all keys.
```

Comment:
> We should add an example. `AsyncIterator` isn't a well known thing.

Makes sense. @superkhau could you please take care of this as part of your work on API docs?